src/commands/mod.rs

```diff
@@ -869,8 +870,19 @@ struct WorkspaceUpdateStaleArgs {}
 
 /// Manage which paths from the working-copy commit are present in the working
 /// copy
+#[derive(Subcommand, Clone, Debug)]
+enum SparseArgs {
+    List(SparseListArgs),
+    Set(SparseSetArgs),
+}
+
+/// List the patterns that are currently present in the working copy
+#[derive(clap::Args, Clone, Debug)]
+struct SparseListArgs {}
+
+/// Update the patterns that are present in the working copy
```

Comment:
This could use some more documentation, like an example of setting and then removing paths in the workspace. As currently `jj sparse --help` is quite empty.

Reply:
Good idea. See how this looks.

Reply:
This is good 👍